backend/db/setup.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Connect to the database and setup it with some default data.
+ */
+"use strict";
+
+const database = require('../db/database.js');
+
+// // This is for setting up database with data from setup.json
+// const setupFile = "setup.json";
+// const fs = require("fs");
+// const path = require("path");
+// const docs = JSON.parse(fs.readFileSync(
+//     path.resolve(__dirname, setupFile),
+//     "utf8"
+// ));
+
+// Do it.
+resetCollection("tickets")
+    .catch(err => console.log(err));
+
+/**
+ * Reset a collection by removing existing content.
+ * Can also be used to insert a default set of documents.
+ *
+ * @async
+ *
+ * @param {string} colName Name of collection.
+ * @param {string=} doc    Documents to be inserted into collection.
```

Comment:
Ta bort?

Reply:
Den är till för att om man vill sätta in bas-data i databasen, t ex om man har en backupfil. Här tänker jag att det är mer flexibelt även om vi just nu inte använder oss av det.